### PR TITLE
useless test: wrote my name for testing purposes as a comment.

### PR DESCRIPTION
**Pull request by mharis2**
pulling from my fork to staging in crowdfly.

### CODE REVIEW

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/MainActivity.java [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    //my name is Haris :p
```
Comment (qasimy123) [inline code/app/src/main/java/com/cmput301w21t06/crowdfly/MainActivity.java:9]: Please remove all emojis -_-

**Review by qasimy123 — CHANGES_REQUESTED** [review]
Good try!

**Review by qasimy123 — APPROVED** [review]

**Comment by mharis2** [comment]
👍

**Review by mu6-Abdullah — APPROVED** [review]
This is cleanish code

**Pull request merged**